src/fetch/stateKey/getStateKeyFromConfig.js

```diff
@@ -4,6 +4,10 @@ import getStateKeyFromUrl from './getStateKeyFromUrl'
 export function getStateKeyFromConfig(config) {
   const { apiPath, url } = config
 
+  if (config.stateKey === null) {
```

Comment:
```suggestion
  if (config.stateKey === null) return null
```
De base, retourner null ou undefined, [c'est pas bien du tout](https://en.wikipedia.org/wiki/Tony_Hoare#Apologies_and_retractions).
Par conséquent, je préfère le moins pire : `null` et le early return qui améliore la lisibilité.